post.md

```diff
@@ -0,0 +1,427 @@
+# Server Actions with User Confirmations in Next.js 13
+
+Next.js 13 has brought exciting new experimental React features to web development. The combination of server side rendered components and server actions have the potential to simplify web development while still giving us the full power of client side React when we need it.
+
+This article introduces the basics of server actions, and how to use them in combination with actions that require user confirmation. This is a common scenario for destructive actions such as deleting a record from a database.
+
+## Sample application
+
+This article presumes you're acquainted with setting up and running Next.js applications.
```

Comment:
Use "assumes" instead of "presumes."

I'd also add a line about how your examples are using Tailwind classes and are in TypeScript.